tests/cql/CqlListOperatorsTest.xml

```diff
@@ -143,6 +143,26 @@
 			<expression>{} = {}</expression>
 			<output>true</output>
 		</test>
+		<test name="EqualABCAndABC">
+			<expression>{ 'a', 'b', 'c' } = { 'a', 'b', 'c' }</expression>
```

Comment:
non-blocking consideration: in general as we build out these tests, do we want to be testing every overload for operators (e.g. for list equality, should we check every overload List<Integer>, List<Decimal>, etc)? Or do we want to adopt some reduced scope convention?